htdocs/editlog.php

```diff
@@ -61,382 +57,470 @@
-                    tpl_set_var('htmlheaders', $headers);
-
-                    if ($use_log_pw == true && $log_pw != '')
-                        if (!$pw_ok && isset($_POST['submitform']))
-                            tpl_set_var('log_pw_field', $log_pw_field_pw_not_ok);
-                        else
-                            tpl_set_var('log_pw_field', $log_pw_field);
-                    else
-                        tpl_set_var('log_pw_field', '');
-
-                    // build smilies
-                    $smilies = '';
-                    if ($descMode != 3)
-                    {
-                        for ($i=0; $i<count($smiley['show']); $i++)
-                        {
-                            if ($smiley['show'][$i] == '1')
-                            {
-                                $tmp_smiley = $smiley_link;
-                                $tmp_smiley = mb_ereg_replace('{smiley_image}', $smiley['image'][$i], $tmp_smiley);
-                                $tmp_smiley = mb_ereg_replace('{smiley_symbol}', $smiley['text'][$i], $tmp_smiley);
-                                $smilies = $smilies.'&nbsp;'.mb_ereg_replace('{smiley_file}', $smiley['file'][$i], $tmp_smiley).'&nbsp;';
-                            }
-                        }
-                        tpl_set_var('smileypath', $opt['template']['smiley']);
+                }
+                $headers .= '<script language="javascript" type="text/javascript" src="' . editorJsPath() . '"></script>' . "\n";
+                tpl_set_var('htmlheaders', $headers);
+
+                if ($use_log_pw == true && $log_pw != '') {
+                    if (!$pw_ok && isset($_POST['submitform'])) {
+                        tpl_set_var('log_pw_field', $log_pw_field_pw_not_ok);
+                    } else {
+                        tpl_set_var('log_pw_field', $log_pw_field);
                     }
-                    tpl_set_var('smilies', $smilies);
+                } else {
+                    tpl_set_var('log_pw_field', '');
                 }
-                else
-                {
-                    //TODO: show error
+
+                // build smilies
+                $smilies = '';
+                if ($descMode != 3) {
+                    for ($i = 0; $i < count($smiley['show']); $i++) {
+                        if ($smiley['show'][$i] == '1') {
+                            $tmp_smiley = $smiley_link;
+                            $tmp_smiley = mb_ereg_replace('{smiley_image}', $smiley['image'][$i], $tmp_smiley);
+                            $tmp_smiley = mb_ereg_replace('{smiley_symbol}', $smiley['text'][$i], $tmp_smiley);
+                            $smilies = $smilies . '&nbsp;' . mb_ereg_replace(
+                                    '{smiley_file}',
+                                    $smiley['file'][$i],
+                                    $tmp_smiley
+                                ) . '&nbsp;';
```

Comment:
Please remove an indent from the lines 488 to 491, as to PSR-2 rule: _Argument lists MAY be split across multiple lines, where each subsequent line is indented **once**._